ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/ip_access_lists.py

```diff
@@ -19,24 +22,29 @@ def ip_access_lists(self) -> list | None:
         """
         Return structured config for ip_access_lists.
         """
-        if not self.shared_utils.is_cv_pathfinder_router:
-            return None
+        ip_access_lists = []
+        if self._svi_acls:
```

Comment:
Please test adding ACL to an SVI that is configured as a pure vlan on an l2leaf. I suspect it will wrongly generate the ACL. I think we should add a check in the `self._svi_acls` function to check for network_services_l3.